### PR TITLE
Proposal pack

### PR DESCRIPTION
**Pull request by jordanlesich**
## Proposal Pack 

- Proposal Card
- Proposal Filter
- Proposal Search
- Proposal List

### CODE REVIEW

File: libs/moloch-v3-macro-ui/tsconfig.lib.json [hunk 1]
```diff
@@ -19,5 +19,11 @@
     "**/*.spec.jsx",
     "**/*.test.jsx"
   ],
-  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
+  "include": [
+    "**/*.js",
+    "**/*.jsx",
+    "**/*.ts",
+    "**/*.tsx",
+    "src/components/ProposalCard"
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/tsconfig.lib.json:27]: any particular reason to list this component in specific?

**Review by santteegt — APPROVED** [review]

File: libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        allowLinks={allowLinks}
+      />
+
+      <ParLg className="title">{proposal.title}</ParLg>
+      <ParMd className="description" color={theme.secondary.step11}>
+        {charLimit(proposal.description, 145)}
+      </ParMd>
+      {isMd && allowLinks && (
+        <StyledRouterLink
+          to={`/molochV3/${daoChain}/${daoId}/proposals/${proposal.proposalId}`}
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx:110]: Router is `/molochV3/${daoChain}/${daoId}/proposal/${proposalid}`

**Review by santteegt — APPROVED** [review]

File: libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+                warning={sensitiveProposalTypes[proposal.proposalType]}
+              >
+                {getProposalTypeLabel(proposal.proposalType, proposalTypes)}
+              </StyledPropType>{' '}
+              | {formatShortDateTimeFromSeconds(proposal.createdAt)}
+            </ParSm>
+          </HeaderContainer>
+          {allowLinks && (
+            <StyledRouterLink
+              to={`/molochV3/${daoChain}/${daoId}/proposals/${proposal.proposalId}`}
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx:262]: Router is `/molochV3/${daoChain}/${daoId}/proposal/${proposalid}`

File: libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx [hunk 4]
```diff
@@ -134,7 +134,7 @@ export const ProposalCardOverview = ({
           {allowLinks && (
             <MemberCardItem asChild>
               <StyledRouterLink
-                to={`/molochv3/${daoChain}/${daoId}/members/${proposal.createdBy}`}
+                to={`/molochV3/${daoChain}/${daoId}/proposal/${proposal.proposalId}`}
```
Comment (santteegt) [inline libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx:137]: @jordanlesich it should be `/molochV3/${daoChain}/${daoId}/member/${proposal.createdBy}`

Comment (jordanlesich) [reply libs/moloch-v3-macro-ui/src/components/ProposalCard/ProposalCardOverview.tsx:137]: Whoops. I'll add that on my next PR.

**Pull request merged**